public/scripts/helper.js

```diff
@@ -0,0 +1,78 @@
+const cardsContainer = document.querySelector('.card_container');
+const category_id = document.querySelector('.category_id'); 
+const flavor_id = document.querySelector('.flavor_id');
+const createCard = (data) => {
+  cardsContainer.textContent = '';
+  data.forEach((element) => {
+    const card = document.createElement('div');
+    card.classList.add('card');
+
+    const cover = document.createElement('img');
+    cover.classList.add('cover');
+    
+    cover.setAttribute('src',element.img_url);
+
+    const name = document.createElement('h3');
+    name.classList.add('name');
+    name.textContent = element.name;
+
+    const quantity = document.createElement('p');
+    quantity.classList.add('quantity');
+    quantity.textContent ="Available Quantity:" + element.quantity;
```

Comment:
To make the code more concise and easier to read, I recommend creating a reusable function to generate the elements.

you can pass in the necessary parameters, such as the element and the class name, and generate the element with the appropriate properties. This will make your code more efficient, easier to maintain, and easier to read.

For example, you could create a function like this:
```js
function createElement(tag, className, textContent) {
  const element = document.createElement(tag);
  element.classList.add(className);
  element.textContent = textContent;
  return element;
}

```